<launch>
  <node name="camera_display" pkg="calibration_node.py" type="sawyer_cam_node"  output="screen" >
    <param name="video_device" value="/dev/video0" />

    <!-- Parameters for Microsoft cameras -->
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />

    <!-- Parameters for Logitech cameras -->
    <!-- <param name="image_width" value="640" /> -->
    <!-- <param name="image_height" value="472" /> -->

    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap" />
  </node>
  
  <!-- Uncomment below to run the image viewer -->
  <!--
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw" />
    <param name="autosize" value="true" />
  </node>
  -->

  <!-- Uncomment below to run the camera service and image processing -->
  <!--
  <node name="camera_srv" pkg="lab4_cam" type="camera_srv.py"  respawn="false" output="screen" />
  <node name="image_process" pkg="lab4_cam" type="image_process.py"  respawn="false" output="screen" />
  -->

</launch>
